code/modules/busy_space/organizations.dm

```diff
@@ -0,0 +1,461 @@
+		"Pioneer",
+		"Hawk",
+		"Haste",
+		"Radiant",
+		"Luminous"
+		)
+	var/list/destination_names = list()	//Names of static holdings that the organization's ships visit regularly.
+	var/autogenerate_destination_names = TRUE
+
+/datum/lore/organization/New()
+	..()
+	if(autogenerate_destination_names) // Lets pad out the destination names.
+		var/i = rand(6, 10)
+		var/list/star_names = list(
+			"Sol", "Alpha Centauri", "Sirius", "Vega", "Regulus", "Vir", "Algol", "Aldebaran",
+			"Delta Doradus", "Menkar", "Geminga", "Elnath", "Gienah", "Mu Leporis", "Nyx", "Tau Ceti",
+			"Wazn", "Alphard", "Phact", "Altair")
+		var/list/destination_types = list("dockyard", "station", "vessel", "waystation", "telecommunications satellite", "spaceport", "distress beacon", "anomaly", "colony", "outpost")
+		while(i)
+			destination_names.Add("a [pick(destination_types)] in [pick(star_names)]")
+			i--
+
+//////////////////////////////////////////////////////////////////////////////////
+
+// TSCs
+/datum/lore/organization/tsc/nanotrasen
+	name = "Nanotrasen Incorporated"
+	short_name = "Nanotrasen"
+	acronym = "NT"
+	desc = "Nanotrasen is one of the foremost research and development companies in SolGov space. \
+	Originally focused on consumer products, their swift move into the field of Plasma has lead to \
+	them being the foremost experts on the substance and its uses. In the modern day, Nanotrasen prides \
+	itself on being an early adopter to as many new technologies as possible, often offering the newest \
+	products to their employees. In an effort to combat complaints about being 'guinea pigs', Nanotrasen \
+	also offers one of the most comprehensive medical plans in SolGov space, up to and including cloning \
+	and therapy.\
+	<br><br>\
+	NT's most well known products are its plasma based creations, especially those used in Cryotherapy. \
+	It also boasts an prosthetic line, which is provided to its employees as needed, and is used as an incentive \
+	for newly tested posibrains to remain with the company."
+	history = "" // To be written someday.
+	work = "research giant"
+	headquarters = "Earth"
+	motto = ""
+
+	ship_prefixes = list("NSV" = "exploration", "NTV" = "hauling", "NDV" = "patrol", "NRV" = "emergency response")
+	// Note that the current station being used will be pruned from this list upon being instantiated
+	destination_names = list(
+		"NSS Exodus in Nyx",
+		"NCS Northern Star in Vir",
+		"NCS Southern Cross in Vir",
+		"NDV Icarus in Nyx",
+		"an asteroid orbiting Kara",
+		"an asteroid orbiting Rota",
+		)
+
+/datum/lore/organization/tsc/nanotrasen/New()
+	..()
+	spawn(1) // BYOND shenanigans means using_map is not initialized yet.  Wait a tick.
+		// Get rid of the current map from the list, so ships flying in don't say they're coming to the current map.
+		var/string_to_test = "[using_map.station_name] in [using_map.starsys_name]"
+		if(string_to_test in destination_names)
+			destination_names.Remove(string_to_test)
+
+
+
```

Comment:
/datum/lore/organization/tsc/donk
	name = "Donk Corporation"
	short_name = "Donk Co."
	acronym = "DC"
	desc = "The infamous rival of the well-known Waffle Corporation, Donk Co. is a company specializing in food delivery systems and brand-name food products such as Donk Pockets. While generally seen as a neutral actor, Donk Corporation has been known to work both with Nanotrasen and the Syndicate when it suits them - often acting as the primary logistical supplier for the Epsilon Eridani sector. Donk Corporation is better known for recent high-profile litigation alleging that their food products are used for illicit drug distribution. While the trial is ongoing, it has been repeatedly delayed due to incidents of methamphetamine poisoning."
	history = ""
	work = "food company that establishes and maintains delivery supply chains"
	headquarters = ""
	motto = "Now with 20% more donk!"

	ship_prefixes = list("D-Co." = "transportation")
	destination_names = list()